bin/setup

```diff
@@ -17,12 +17,13 @@ FileUtils.chdir APP_ROOT do
   system! "gem install bundler --conservative"
   system("bundle check") || system!("bundle install")
 
-  # Install JavaScript dependencies
-  system! 'bin/yarn'
-
```

Comment:
Just want to check that this is ok to remove?

Reply:
This line was added by 2963c7147490bd5e83d44fbc67943a2080c58a40 with the commit message: "Apply rails app:update changes", so it looks like a change suggested by app:update and not a separately introduced thing. The current app:update run suggests to remove it. I followed the suggestion.

It might be related to decoupling Rails from JavaScript in the newer versions, as I think there was a period when the direction of Rails was supposedly to integrate more tightly with JavaScript package managers etc. but nowadays Hotwire seems to be the direction.

What's your opinion on that?

Reply:
Sounds good to me! I'll approve